test/viper/invariant.mo

```diff
@@ -4,10 +4,13 @@ actor {
 
   var count = 0 : Int;
 
-  assert claimed and not (-1 == -1) and (-42 == -42) or true;
-  assert count > 0;
+  assert:invariant claimed and not (-1 == -1) and (-42 == -42) or true;
```

Comment:
Of course we can change this to just be `invariant`.

Reply:
This looks much better, thanks!